xonsh/xontribs.py

```diff
@@ -78,8 +77,34 @@ def update_context(name, ctx=None):
     return ctx.update(modctx)
 
 
-def xontribs_load(names, verbose=False):
-    """Load xontribs from a list of names"""
+class XontribCompleter(ArgCompleter):
```

Comment:
Maybe only complete non-loaded xontribs?

Reply:
yep a good idea.

Reply:
I will be creating separate PRs for all these builtins aliases, once the base class is merged. I will take into account all the other comments in them.

Reply:
done locally. collapsing to remind myself